docs/src/usage/std.md

```diff
@@ -67,15 +67,22 @@ This field is used for allowing smarter introspection of how the argument given
 - "read" - This argument is only read from. Currently unused.
 - "write" - This argument is only written to. Used by `unused_variable` to assist in detecting a variable only being written to, even if passed into a function.
 
+#### "must_use"
+This field is used for checking if the return value of a function is used.
+
+- `true` - The default. The return value of this function must be used.
+- `false` - The return value of this function does not need to be used.
+
 Example:
 ```yml
- table.insert:
```

Comment:
Why remove the example for `observes`?

Reply:
Could I just give a example for each one instead? I can't think of a native function of Luau that satifies all these.

Reply:
Yes that’s what I mean. In your diff you removed the example for the previous section. Keep the previous one and make a new example for `must_use`.

Reply:
Resolved in: https://github.com/Kampfkarren/selene/pull/583/commits/607ca211d56b7850df50006631eceb5ebc9c87a6